GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -138,6 +138,10 @@ public CommitInfo()
                     .Subscribe(_ => handler(_.EventArgs));
 
             commitInfoHeader.SetContextMenuStrip(commitInfoContextMenuStrip);
+
+            // at this point rtbxCommitMessage.Bounds = {X = 8 Y = 8 Width = 440 Height = 0}
+            // and with Height=0 we won't be receiving any ContentsResizedEvents
+            rtbxCommitMessage.Height = 1;
```

Comment:
💭 Just to make sure, this was intentional?

Reply:
Yes, as per the comment. Setting the height to a positive number makes it visible, and it starts triggering resize event.